Kernel/Thread.h

```diff
@@ -381,7 +382,7 @@ class Thread {
     void set_default_signal_dispositions();
     void push_value_on_stack(FlatPtr);
 
-    u32 make_userspace_stack_for_main_thread(Vector<String> arguments, Vector<String> environment);
+    u32 make_userspace_stack_for_main_thread(Vector<String> arguments, Vector<String> environment, Vector<AuxiliaryValue> auxv);
```

Comment:
I would omit the last parameter name here since it's obvious from the type.

Reply:
True, originally I had a `Vector<auxv_t>` which wasn't quite as obvious.